File: Front/src/app/models/pregunta.ts
import { Respuesta } from "./respuesta";
export class Pregunta{
    descripcion:string;
    respuestas:Respuesta[];
    hide?:boolean;
    
    constructor(descripcion:string,respuesta:Respuesta[]) {
        this.descripcion = descripcion;
        this.respuestas = respuesta;
        this.hide = true;
    }
}

File: BackEnd/BackEnd/Domain/Model/Pregunta.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace BackEnd.Domain.Model
{
    public class Pregunta
    {
        public int? Id { get; set; }
        [Required]
        [Column(TypeName = "varchar(150)")]
        public string Descripcion { get; set; }
        [Required]
        public List<Respuesta> Respuestas { get; set; }
        public int? CuestionarioId { get; set; }
        public Cuestionario? Cuestionario{ get; set; }
    }
}
